packages/ember-routing/lib/services/router.js

```diff
@@ -6,6 +6,23 @@ import { shallowEqual, resemblesURL, extractRouteArgs } from '../utils';
    The Router service is the public API that provides component/view layer
    access to the router.
 
+   A simple example of usage:
```

Comment:
Can you be more descriptive and say what the example is showing?

Reply:
I updated the PR to expand the intro and add an description to the example. 
I reused some text from the original RFC [RFC095](https://github.com/emberjs/rfcs/blob/master/text/0095-router-service.md).
@locks what do you think?